### PR TITLE
added testcase to detect default method usage

### PR DESCRIPTION
**Pull request by ivadimko**

### CODE REVIEW

File: src/arrayMethodJoin.test.js [hunk 1]
```diff
@@ -9,6 +9,11 @@ test('join2 is added to [].__proto__', () => {
     .toBeInstanceOf(Function);
 });
 
+test(`join2 doesn't call default join`, () => {
+  expect([].join2.toString().includes('.join'))
+    .toBe(false);
+});
```
Comment (mgrinko) [inline src/arrayMethodJoin.test.js]: it is overkill
I think we believe they won't use it

Comment (ivadimko) [reply src/arrayMethodJoin.test.js]: https://github.com/mate-academy/js_array-method-push/pull/5/files

Comment (mgrinko) [reply src/arrayMethodJoin.test.js]: Ok let's add the test

Comment (mgrinko) [reply src/arrayMethodJoin.test.js]: @ivadimko lets check for `.join(`

**Pull request merged**